### ISSUE TITLE
slurmctld and slurmdbd requires restart after ingesting ColdFront data

### ISSUE THREAD

**Issue by tomgreen66** [ubccr/hpc-toolset-tutorial#109]
I just following the tutorial and find that the user cgray cannot submit a job and I get:

```
[cgray@frontend ~]$ sbatch --wrap "sleep 60"
sbatch: error: Batch job submission failed: Invalid account or account/partition combination specified
```

After restarting slurmctld and slurmdbd docker containers it works:

```
[cgray@frontend ~]$ sbatch --wrap "sleep 60"
Submitted batch job 2
```

It seems slurmctld doesn't recognise the user until after the restart - I thought it might be something sssd related but restart seems to do the trick.  Is this a known issue?

slurmctld log shows for failure:
```
[2021-11-04T13:34:21.562] error: User 1002 not found
[2021-11-04T13:34:21.562] _job_create: invalid account or partition for user 1002, account '(null)', and partition 'comp
ute'
[2021-11-04T13:34:21.562] _slurm_rpc_submit_batch_job: Invalid account or account/partition combination specified
[2021-11-04T13:40:08.213] error: User 1002 not found
[2021-11-04T13:40:08.213] _job_create: invalid account or partition for user 1002, account '(null)', and partition 'comp
ute'
[2021-11-04T13:40:08.213] _slurm_rpc_submit_batch_job: Invalid account or account/partition combination specified
```

For successful run:
```
[2021-11-04T13:40:16.443] _slurm_rpc_submit_batch_job: JobId=2 InitPrio=4294901759 usec=606
[2021-11-04T13:40:17.046] SchedulerParameters=default_queue_depth=100,max_rpc_cnt=0,max_sched_time=2,partition_job_depth=0,sched_max_job_start=0,sched_min_interval=2
[2021-11-04T13:40:20.048] sched: Allocate JobId=2 NodeList=cpn01 #CPUs=1 Partition=compute
[2021-11-04T13:41:20.242] _job_complete: JobId=2 WEXITSTATUS 0
[2021-11-04T13:41:20.242] _job_complete: JobId=2 done
```

**Comment by aebruno** [comment]
@tomgreen66 Thanks for reporting this. What OS/Docker version are you running?

**Comment by tomgreen66** [comment]
Hopefully this is enough?

```
$ cat /etc/centos-release
CentOS Linux release 7.9.2009 (Core)
```

```
$ docker --version
Docker version 20.10.10, build b485636
```

**Comment by aebruno** [comment]
@dsajdak Let's check and see if this is still an issue in the new container builds for PEARC22.

**Comment by aebruno** [comment]
I'm guessing this is also related to #119. This should hopefully be fixed in #123
